src/core/index.js

```diff
@@ -134,10 +140,12 @@ export const createExecuteCommand = ({
       componentRegistry,
       getImmediatelyAvailableTools(componentName) {
         const componentLogger = createComponentLogger(componentName);
+        const fireReferrerHideableImage = injectFireReferrerHideableImage();
```

Comment:
If you put this above the `createExecuteCommand` export, we can share the iframe across instances of Web SDK.